### PR TITLE
Update Theme READMEs with more explicit instructions

### PR DESCRIPTION
**Pull request by laurieontech**
When walking through theme setup I noticed that the READMEs would benefit from additional instructions for those not using a starter. This PR addresses the READMEs for the blog, blog core, and notes theme.

### CODE REVIEW

**Comment by bball07** [comment]
These instructions are awesome!! Great job @laurieontech :)

File: packages/gatsby-theme-blog-core/README.md [hunk 1]
```diff
@@ -11,30 +11,55 @@ A Gatsby theme for creating a blog child theme. It includes all of the data stru
...
+    },
+  ],
+}
 ```
 
+3. Add blog posts to your site by creating `md` or `mdx` files inside `/content/posts`.
+
+   > Note that if you've changed the default `contentPath` in the configuration, you'll want to add your markdown files in the directory specified by that path.
+
+4. Run your site using `gatsby develop` and navigate to your notes. If you used the above configuration, your URL will be `http://localhost:8000/blog`
```
Comment (gillkyle) [inline packages/gatsby-theme-blog-core/README.md]:
```suggestion
4. Run your site using `gatsby develop` and navigate to your posts. If you used the above configuration, your URL will be `http://localhost:8000/blog`
```

was this copy/pasted from the steps on the notes theme maybe? Seems like maybe it should say something like blog posts?

File: packages/gatsby-theme-blog/README.md [hunk 2]
```diff
@@ -11,32 +11,59 @@ A Gatsby theme for creating a blog.
...
+    },
+  ],
+}
+```
+
+3. Add blog posts to your site by creating `md` or `mdx` files inside `/content/posts`.
+
+   > Note that if you've changed the default `contentPath` in the configuration, you'll want to add your markdown files in the directory specified by that path.
+
+4. Run your site using `gatsby develop` and navigate to your notes. If you used the above configuration, your URL will be `http://localhost:8000/blog`
```
Comment (gillkyle) [inline packages/gatsby-theme-blog/README.md]: Same thing here?

```suggestion
4. Run your site using `gatsby develop` and navigate to your blog posts. If you used the above configuration, your URL will be `http://localhost:8000/blog`
```

**Comment by laurieontech** [comment]
Thanks @gillkyle! SHould be fixed.

**Pull request merged**